scripts/bench-single-owner-multikey.py

```
@@ -0,0 +1,235 @@
+        b = p.encode() if isinstance(p, str) else p
+        out += b"$%d\r\n%s\r\n" % (len(b), b)
+    return out
+
+def spawn(binary, port, dirpath):
+    p = subprocess.Popen([binary, "--port", str(port), "--shards", SHARDS,
+                          "--appendonly", "no", "--disk-free-min-pct", "0",
+                          "--dir", dirpath],
+                         stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+    for _ in range(300):
+        try:
+            s = socket.create_connection(("127.0.0.1", port), 0.2); s.close(); return p
```

Ruff (0.16.2)

[error] Multiple statements on one line (semicolon)

(E702)